src/rapids_dependency_file_generator/_constants.py

```diff
@@ -3,7 +3,6 @@
 default_channels = [
     "rapidsai",
     "rapidsai-nightly",
-    "dask/label/dev",
     "conda-forge",
     "nvidia",
```

Comment:
We can also drop `nvidia` which is only needed for CUDA 11. See references here:
- https://github.com/rapidsai/gha-tools/pull/146 for `nvidia`
- https://github.com/rapidsai/gha-tools/pull/157 for `dask/label/dev`
```suggestion
```